packages/jamf_protect/data_stream/alerts/manifest.yml

```
@@ -0,0 +1,57 @@
+title: Receives Alerts from Jamf Protect with Elastic Agent.
+type: logs
+streams:
+  - input: http_endpoint
+    template_path: http_endpoint.yml.hbs
+    title: Jamf Protect Alerts
+    description: Receives Alerts from Jamf Protect with Elastic Agent.
+    vars:
+      - name: listen_port
+        type: integer
+        title: Listen Port
+        description: The port number the listener binds to.
+        multi: false
+        required: true
+        show_user: true
+        default: 9551
+      - name: url
+        type: text
+        title: URL
+        description: This option specifies which URL path to accept requests on. Defaults to /.
+        multi: false
+        required: false
+        show_user: false
+        default: /
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: false
+        default:
+          - forwarded
+          - jamfprotect_alerts
```

```suggestion
          - jamf_protect-alerts
```
As a convention. Please change for other datastreams as well.